gradle/mod.gradle

```diff
@@ -1161,6 +1161,22 @@ tasks.withType(Jar.class, {task ->
     task.duplicatesStrategy 'exclude'
 })
 
+tasks.register("outputChangelogHeader") {
+    def changelog = new java.io.File("changelog.md")
```

Comment:
I really have feeling this should be "file" but Thodor insisted
```suggestion
    def changelog = file("changelog.md")
```

Reply:
This needs to indeed be file.

Reply:
Maybe useful if you state the reason. This needs to adhere to cwd, not project root relative.

Reply:
No actually, i needs to adhere to the project directory.
That is a core concept within gradle projects.